Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MKMapViewTests.swift
+//  SwifterSwift
+//
+//  Created by Hannes Staffler on 24.01.19.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(MapKit)
+import MapKit
+
+import struct CoreLocation.CLLocationCoordinate2D
+
+final class MKMapViewTests: XCTestCase {
```

Comment:
The conditional compilation and attributes should match the source file.
```suggestion
#if !os(watchOS)
@available(iOS 3.0, tvOS 9.2, macOS 10.9, *)
final class MKMapViewTests: XCTestCase {
```
Don't forget the closing `#endif`